src/components/atoms/HookFormInputWrapper/HookFormInputWrapper.tsx

```diff
@@ -44,6 +46,7 @@ export default function HookFormInputWrapper({
             onChangeValue,
             ref,
             value,
+            ...childProps,
```

Comment:
```
<HookFormInputWrapper name={name}>
    <NumberInput label={name} size="small" variant="material" {...props} />
</HookFormInputWrapper>
```
when passing children like above, all the props of the child (`label`, `size`, etc.) are retained, so a little unclear of the usage of this new prop, can you provide an example?

Reply:
the error object is passed directly to the children inside the controller. hence, the error will automatically be shown. Now, If we don't want to use the error to be automatically shown (eg. sneak-peak component in teaser) then we have to see the default error to undefined by passing childprops as {error: undefined}

`<HookFormInputWrapper
                name="age"
                control={control}
                childProps={{ error: undefined }}
              >
                <BasicNumberInput
                  format
                  parse
                  value={inputData.age}
                  className={styles.age}
                />
              </HookFormInputWrapper>`